### PR TITLE
Patch to show text in dropdowns

### PR DESCRIPTION
**Pull request by crackjack**

### CODE REVIEW

File: smart_selects/static/smart-selects/admin/js/chainedfk.js [hunk 1]
```diff
@@ -47,6 +47,8 @@
                     return;
                 }
                 $.getJSON(url, function (j) {
+                    // a quick & dirty hack to get the values appear in dropdown selection
+                    j = j.response;
```
[the remark this answers is not in the document]
Comment (blag) [reply smart_selects/static/smart-selects/admin/js/chainedfk.js]: Nevermind, I see your explanation from [here](https://github.com/digi604/django-smart-selects/issues/56#issuecomment-379360369). Sorry for the noise, I hadn't dug through my inbox enough. 😆

[the remark this answers is not in the document]
Comment (crackjack) [reply smart_selects/static/smart-selects/admin/js/chainedfk.js]: is the PR accepted? I am also working on another issue where this doesn't work during edit.

[the remark this answers is not in the document]
Comment (blag) [reply smart_selects/static/smart-selects/admin/js/chainedfk.js]: Not until I can figure out where the `response` and `errors` keys in the returned JSON are coming from. They don't appear to be coming from this package, as far as I can tell.

[the remark this answers is not in the document]
Comment (crackjack) [reply smart_selects/static/smart-selects/admin/js/chainedfk.js]: they are a part of the HttpResponse. The HttpResponse method returns all those, try these:
```
def abc(request):
    return HttpResponse(json.dumps({'hello': 'world'}), content_type='application/json')
```

and the output would be
```
{"errors": [], "response": {"hello": "world"}, "server": "localhost:8000"}
```

[the remark this answers is not in the document]
Comment (crackjack) [reply smart_selects/static/smart-selects/admin/js/chainedfk.js]: this is the exact line in code if you'd like https://github.com/digi604/django-smart-selects/blob/master/smart_selects/views.py#L90

I really hope you could merge these 2 PR and publish in pypi.

[the remark this answers is not in the document]
Comment (crackjack) [reply smart_selects/static/smart-selects/admin/js/chainedfk.js]: hey @blag 

please review these if you have some spare time

[the remark this answers is not in the document]
Comment (blag) [reply smart_selects/static/smart-selects/admin/js/chainedfk.js]: @crackjack I don't get that response:

```python
import json
from django.http import HttpResponse

print(HttpResponse(json.dumps({'hello': 'world'}), content_type='application/json').content)
# b'{"hello": "world"}'
```

This is with Django 2.0.0.

Where in the **Django** source code does it add the `response` and `errors` keys?

**Comment by basilrabi** [comment]
This patch works for me. Thank you.

**Comment by manelclos** [comment]
@crackjack can you please against latest master to see if this PR is still necessary?

**Comment by manelclos** [comment]
Closing inactive PR, please reopen if necessary.

**Pull request closed without merging**